source/includes/fundamentals/code-snippets/BulkWrite.java

```diff
@@ -77,15 +77,16 @@ private void insertExceptionExample() {
         // begin insertExceptionExample
         try {
             List<WriteModel<Document>> bulkOperations = new ArrayList<>();
-            
             InsertOneModel<Document> doc1 = new InsertOneModel<>(new Document("_id", 1));
             InsertOneModel<Document> doc3 = new InsertOneModel<>(new Document("_id", 3));
             
             bulkOperations.add(doc1);
             bulkOperations.add(doc3);
             
+            // Runs a bulk write operation for the specified insert WriteModels
```

Comment:
Suggestion:
For consistency, I think the InsertOneModel lines should also be commented similar to the ones in bulkWriteNotOrderedExample.

Reply:
Suggestion:
For consistency, include whether ordered. The unordered case is shown in the [code comments sheet](https://docs.google.com/spreadsheets/d/1gCIeV5Bsp1G69MUNQZm8bYK9UNVSQjJt9T7eUM8aJiE/edit#gid=0):

```suggestion
            // Runs a bulk write operation for the specified insert WriteModels in order
```

Applies to all instances.